app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryEffectHandler.kt

```diff
@@ -68,9 +71,19 @@ class BloodSugarEntryEffectHandler @AssistedInject constructor(
         .addConsumer(SetBloodSugarReading::class.java, { ui.setBloodSugarReading(it.bloodSugarReading) }, schedulersProvider.ui())
         .addTransformer(UpdateBloodSugarEntry::class.java, updateBloodSugarEntryTransformer(schedulersProvider.io()))
         .addConsumer(ShowConfirmRemoveBloodSugarDialog::class.java, { ui.showConfirmRemoveBloodSugarDialog(it.bloodSugarMeasurementUuid) }, schedulersProvider.ui())
+        .addTransformer(LoadBloodSugarUnitPreference::class.java, loadBloodSugarUnitPreference())
         .build()
   }
 
+  private fun loadBloodSugarUnitPreference(): ObservableTransformer<LoadBloodSugarUnitPreference, BloodSugarEntryEvent> {
+    return ObservableTransformer { effects ->
+      effects
+          .observeOn(schedulersProvider.io())
+          .map { bloodSugarUnitPreference.get() }
```

Comment:
This is not an observable stream, we want this to be an observable stream so that whenever the unit changes we can update the layout accordingly.

Reply:
So currently it's an immediate call and it will only send out `BloodSugarUnitPreferenceLoaded` once whenever the effect is called.